terraform/modules/sync_check/service/sync_check.rb

```diff
@@ -51,8 +51,8 @@ def get_and_assert_env_variable(name)
   client.post_message "✅ Sync check for #{hostname} passed. 🌲🌳🌲🌳🌲"
 else
   client.post_message "⛔ Sync check for #{hostname} fiascoed. 🔥🌲🔥"
-  SyncCheck.new.run_forest_cli("--chain #{network} db clean --force")
-  logger.info 'DB Cleaned'
+  SyncCheck.new.run_forest_cli("db destroy --chain #{network} --force")
```

Comment:
What command will it invoke in the end? Are you it's what you want?

Reply:
It will do the same than before, the subcommand `forest-cli clean` was rename to `forest-tool destroy`.

Reply:
is it `run_forest_cli` or `run_forest_tool`? :)